### PR TITLE
Ibeat support for new vct parameter for input

### PR DESCRIPTION
**Pull request by denmark-admin**

### CODE REVIEW

File: extensions/amp-analytics/0.1/test/vendor-requests.json [hunk 1]
```diff
@@ -600,4 +600,4 @@
     "mediaPosition": "https://!trackDomain/!trackId/wt?p=440,_title_,1,_screen_width_x_screen_height_,_screen_color_depth_,1,,_document_referrer_,_viewport_width_x_viewport_height_,0&tz=_timezone_&eid=_client_id(amp-wt3-eid)_&la=_browser_language_&mi=!id&mk=pos&mt1=!currentTime&mt2=!duration&&pu=_source_url_&eor=1&x=!playedTotal",
     "mediaEnded": "https://!trackDomain/!trackId/wt?p=440,_title_,1,_screen_width_x_screen_height_,_screen_color_depth_,1,,_document_referrer_,_viewport_width_x_viewport_height_,0&tz=_timezone_&eid=_client_id(amp-wt3-eid)_&la=_browser_language_&mi=!id&mk=eof&mt1=!currentTime&mt2=!duration&&pu=_source_url_&eor=1&x=!playedTotal"
   }
-}
+}
```
Comment (alanorozco) [inline extensions/amp-analytics/0.1/test/vendor-requests.json:603]: Please restore newline.

File: extensions/amp-analytics/0.1/vendors/ibeatanalytics.json [hunk 2]
```diff
@@ -13,4 +13,4 @@
       "request": "track"
     }
   }
-}
+}
```
Comment (alanorozco) [inline extensions/amp-analytics/0.1/vendors/ibeatanalytics.json:16]: Please restore newline.

**Review by alanorozco — APPROVED** [review]
Looks good to me. Thank you!

File: extensions/amp-analytics/0.1/test/vendor-requests.json (hunk 1 above)
Comment (denmark-admin) [reply extensions/amp-analytics/0.1/test/vendor-requests.json]: newline restored

**Comment by denmark-admin** [comment]
@alanorozco , this branch updated from `main` branch

**Comment by alanorozco** [comment]
@denmark-admin Thanks! Merged

**Pull request merged**